fissionServerSwagger.yaml

```diff
@@ -0,0 +1,386 @@
+openapi: 3.1.0
+info:
+  title: Fission Server
+  description: |-
+    This is a sketch of the API for the Fission-Server-in-Rust.
+
+    Some useful links:
+    - [The Fission Server repository](https://github.com/fission-codes/fission-server)
+    
+  version: 3.0.0alpha
+
+tags:
+  - name: auth
+    description: Authentication
+  - name: account
+    description: Account management
+  - name: volume
+    description: Volume management
+  - name: app
+    description: App management
+  - name: domain
+    description: Domain (DNS) management
+
+paths:
+    /auth/requestToken:
+      post:
+        tags:
+          - account
+          - auth
+        summary: Request an email verification token.
+        description: |-
+          Reqeust an email verification token, to be used to [create an account](#/paths/~1account~1create/post) or [reset the DID](#/paths/~1account~1update/post) associated with an account.
+        requestBody:
+          description: Email address
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  email:
+                    $ref: "#/components/schemas/Account/properties/email"
+        responses:
+          '200':
+            description: success
+          '400':
+            description: Bad Request
+          '429':
+            description: Too Many Requests
+          '500':
+            description: Internal Server Error
+
+    /account/create:
```

Comment:
Nitpick, maybe just make this `POST /account` to line up with the other endpoints? Or maybe even `POST /account/{accountName}`.